src/js/trove/table.js

```diff
@@ -515,9 +515,9 @@
           return runtime.ffi.makeList(headers);
         }),
 
-        'stack': runtime.makeMethod1(function(_, otherTable) {
+        'stack': runtime.makeMethod1(function(self, otherTable) {
           ffi.checkArity(2, arguments, "stack", true);
-          runtime.checkArgsInternal1("tables", "stack", otherTable, annTable);
+          runtime.checkArgsInternal1("tables", "stack", self, annTable, otherTable, annTable);
```

Comment:
It occurs to me that we could have a checkMethodArgsInternal1-N that does this check, doesn't check the `self` param (assuming currying worked out), and gets the arity right.

Of course, since in the current compiler the `self` _could_ be the wrong type because of detached currying, this is necessary.